examples/ring_with_tracing/deps.edn

```diff
@@ -0,0 +1,22 @@
+{:paths ["src" "resources"]
+
+ :deps {org.clojure/clojure {:mvn/version "1.10.3"}
+        ring/ring-core {:mvn/version "1.9.5"}
+        ring/ring-jetty-adapter {:mvn/version "1.9.5"}
+        ring/ring-json {:mvn/version "0.5.1"}
+        integrant/integrant {:mvn/version "0.8.0"}
+        integrant/repl {:mvn/version "0.3.2"}
+        ;;
+        ;;
```

Comment:
Hi!

May I suggest that this is redone, following the layout of examples/basic/deps.edn. It has clojure library at the top, then everything follows the 3 lines of semicolons, sorted.

Reply:
OK. I've fixed.

Reply:
OK. I'll fix it.